coalib/results/result_actions/IgnoreResultAction.py

```diff
@@ -46,22 +47,32 @@ def apply(self, result, original_file_dict, file_diff_dict, language: str,
         """
         Add (I)gnore comment
         """
+        source_range = next(filter(lambda sr: exists(sr.file),
+                                   result.affected_code))
+        filename = source_range.file
 
-        ignore_comment = self.get_ignore_comment(result.origin, language)
+        original_line = original_file_dict[filename][
+                        source_range.start.line-1].rstrip()
```

Comment:
this is incorrect indentation.  `source_range` is not at the same scope level as `original_file_dict`